oak/server/rust/oak_runtime/src/node/grpc_server.rs

```diff
@@ -94,18 +95,20 @@ impl GrpcServerNode {
     /// their values after the gRPC server pseudo-Node has started and a separate thread was
     /// initialized.
     pub fn new(
-        config_name: &str,
+        display_name: &str,
+        config: GrpcServerConfiguration,
         runtime: RuntimeProxy,
-        address: SocketAddr,
         initial_reader: Handle,
-    ) -> Self {
-        Self {
-            config_name: config_name.to_string(),
+    ) -> Result<Self, ConfigurationError> {
```

Comment:
Is `new` supposed to return `Result` instead of just `Self`.
Should this function be called `create` instead? WDYT?

Reply:
I am also not quite sure about naming, but I think given the return type is obvious, it should be fine either way.